Calculations.py

```diff
@@ -11,57 +11,51 @@ def math_model(all_characters, max_stats, promotion_gains, base_classes, promote
 	'''
 	
 	expected_stats = []
+	index = -1
+	count = 0
+	for character in all_characters:
+		if(character.getName() == char):
+			index = count
+		count += 1
+	if(index == -1):
```

Comment:
Since this won't be controlled by the user, an error like this will be a huge issue. We will have to figure out a way to resolve this. Just printing an error won't be the correct way to handle it. But, this is okay for now